pcli/src/command/tx/proposal.rs

```diff
@@ -54,6 +54,22 @@ pub enum ProposalCmd {
         #[clap(long)]
         source: Option<u64>,
     },
+    /// Claim a governance proposal deposit for a proposal you submitted that has finished voting.
+    ///
+    /// This consumes the voting or withdrawn proposal NFT and mints an NFT representing whether the
+    /// proposal passed, failed, or was vetoed. In the case of a veto, the deposit is not returned
+    /// by this action; in other cases, it is returned to you.
+    DepositClaim {
+        /// The transaction fee (paid in upenumbra).
+        #[clap(long, default_value = "0")]
+        fee: u64,
+        /// The proposal id to claim the deposit for.
+        #[clap(long = "on")]
+        proposal_id: u64,
+        /// Optional. Only spend funds originally received by the given address index.
+        #[clap(long)]
+        source: Option<u64>,
+    },
```

Comment:
A new `pcli` command to claim a proposal deposit. This does the lookups behind the scenes to find out what outcome the proposal had and what the deposit is, so you don't have to look them up and feed them into the command (this can't be integrated into the planner as it currently is, because these queries require the specific query service).